### PR TITLE
Clarify description for `viewId` argument in `vscode.openWith` command

### PR DESCRIPTION
**Pull request by jruales**
Just changing the comment string for `openWith` API to make it clear what `viewId` means, since it's referred to as `viewType` elsewhere.

### CODE REVIEW

File: src/vs/workbench/api/common/extHostApiCommands.ts [hunk 1]
```diff
@@ -394,7 +394,7 @@ const newCommands: ApiCommand[] = [
 		'vscode.openWith', '_workbench.openWith', 'Opens the provided resource with a specific editor.',
 		[
 			ApiCommandArgument.Uri.with('resource', 'Resource to open'),
-			ApiCommandArgument.String.with('viewId', 'Custom editor view id or \'default\' to use VS Code\'s default editor'),
+			ApiCommandArgument.String.with('viewId', 'Custom editor view id. This should match the viewType from the customEditors contribution point. Use \'default\' to use VS Code\'s default editor'),
```
Comment (jrieken) [inline src/vs/workbench/api/common/extHostApiCommands.ts:397]: This is custom editors but more than just that, there is also notebooks and likely others. Adding @lramos15 for the precise set of possible groups of view ids

**Review by jrieken — CHANGES_REQUESTED** [review]

File: src/vs/workbench/api/common/extHostApiCommands.ts (hunk 1 above)
Comment (lramos15) [reply src/vs/workbench/api/common/extHostApiCommands.ts:397]: The main three I would document here is similar to the tabs api documentation. It's notebook's `notebookType` , custom editor's `viewId`, and for the text editor it's `default`

Comment (jruales) [reply src/vs/workbench/api/common/extHostApiCommands.ts:397]: @lramos15 The confusing part is that custom editors don't have something that's publicly called `viewId`. In all the other API documentation for creating a custom editor, it's called `viewType` and not `viewId`, right?

Comment (jruales) [reply src/vs/workbench/api/common/extHostApiCommands.ts:397]: image

Comment (lramos15) [reply src/vs/workbench/api/common/extHostApiCommands.ts:397]: @jruales Yes `viewType` not `viewId` sorry. We have a mixed bag of words for this everywhere which is a bit annoying but `viewType`, `notebookType`, and `default` are the main things that can be passed into `viewId`

Comment (jruales) [reply src/vs/workbench/api/common/extHostApiCommands.ts]: @jrieken  I've updated the PR based on the feedback

**Review by lramos15 — APPROVED** [review]

**Review by jrieken — APPROVED** [review]

**Pull request merged**